configFiles/pipeline_config onnxRuntime.yaml

```
@@ -0,0 +1,399 @@
+                backend: onnxruntime
+          TextRecognition:
+            module_name: text_recognition
+            model_name: PP-OCRv5_server_rec
+            model_dir: null
+            use_hpip: True
+            hpi_config:
+                backend: onnxruntime
+            batch_size: 8
+        score_thresh: 0.0
+
+  SealRecognition:
```

YAMLlint (1.37.1)

[error] duplication of key "SealRecognition" in mapping

(key-duplicates)